testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -111,6 +113,20 @@ impl EventObserver {
         }
     }
 
+    fn make_new_microblocks_payload(receipts: Vec<StacksTransactionReceipt>) -> serde_json::Value {
+        let mut tx_index = 0;
+        let mut serialized_txs = Vec::new();
+        for receipt in receipts.iter() {
+            let payload = EventObserver::make_new_block_txs_payload(receipt, tx_index);
+            serialized_txs.push(payload);
+            tx_index += 1;
+        }
+
+        json!({
+            "transactions": serialized_txs,
```

Comment:
We'll also need the transaction events -- I think the type is `StacksTransactionEvent`.